lib/terraforming/resource/iam_role.rb

```diff
@@ -16,25 +16,25 @@ def initialize(client)
       end
 
       def tf
-        apply_template(@client, "tf/iam_role")
+        apply_template(@client, 'tf/iam_role')
       end
 
       def tfstate
         iam_roles.inject({}) do |resources, role|
           attributes = {
-            "arn" => role.arn,
-            "assume_role_policy" =>
+            'arn' => role.arn,
+            'assume_role_policy' =>
               prettify_policy(role.assume_role_policy_document, breakline: true, unescape: true),
-            "id" => role.role_name,
-            "name" => role.role_name,
-            "path" => role.path,
-            "unique_id" => role.role_id,
+            'id' => role.role_name,
+            'name' => role.role_name,
+            'path' => role.path,
+            'unique_id' => role.role_id,
```

Comment:
__[rubocop]__
* Avoid comma after the last item of a hash. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas)